db/schema.yml

```diff
@@ -4541,6 +4541,17 @@ middleware_domains:
 - ems_id
 - created_at
 - updated_at
+middleware_messagings:
```

Comment:
I don't think you're supposed to commit schema.yml, it's an auto generated file. @jrafanie ?